### PR TITLE
Release 2023.04.2 

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Release notes added for weekly release.
- **Is this PR related to a Monolith release?** 2023.04.2

### Resolves:

* [OKTA-600440](https://oktainc.atlassian.net/browse/OKTA-600440)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/d89bc79ed06596679aa7a60bb7564d762486ea6c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3b77d02d-e2fc-435e-b678-9d215515ddf2) | 60  | 80  | 47  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/d89bc79ed06596679aa7a60bb7564d762486ea6c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | [72](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c4b645ec-f32d-4566-b707-05e9460a4cf0) | 75  | 81  | 44  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.ea9c289c-9c67-4065-8f2f-d1dba7280d9d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 1]
```diff
@@ -6,6 +6,26 @@ title: Okta Identity Engine API Products release notes 2023
...
+| ------ | ------------------------ |
+| [Bugs fixed in 2023.04.2](#bugs-fixed-in-2023-04-2) | April 26, 2023 |
+
+#### Bugs fixed in 2023.04.2
+
+- When the sign-in page was edited using the code editor, the event type `system.custom_error.update` was logged. (OKTA-591800)
+
+- The Policy API `constraints` object could be set with the `types` and `methods` of disabled authenticators or methods. (OKTA-586221)
+
+- A "Next" link wasn't returned when a List Identity Providers request was made and a limit of 200 was set. (OKTA-597359)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:21]:
```suggestion
- A **Next** link wasn't returned when a List Identity Providers request was made and a limit of 200 was set. (OKTA-597359)
```
Not sure if we're supposed to use quotes?

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md [hunk 2]
```diff
@@ -4,6 +4,20 @@ title: Okta API Products release notes 2023
...
+
+| Change | Expected in Preview Orgs |
+| ------ | ------------------------ |
+| [Bugs fixed in 2023.04.2](#bugs-fixed-in-2023-04-2) | April 26, 2023 |
+
+#### Bugs fixed in 2023.04.2
+
+- When the sign-in page was edited using the code editor, the event type `system.custom_error.update` was logged. (OKTA-591800)
+
+- A "Next" link wasn't returned when a List Identity Providers request was made and a limit of 200 was set. (OKTA-597359)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023/index.md:17]:
```suggestion
- A **Next** link wasn't returned when a List Identity Providers request was made and a limit of 200 was set. (OKTA-597359)
```

**Review by vanngo-okta — APPROVED** [review]
Small style nit. Otherwise, LGTM! 🚀

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md (hunk 1 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:21]: Yeah, I wondered about that. I think I'll use code font: https://developer.okta.com/docs/reference/core-okta-api/#link-header

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md (hunk 2 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023/index.md:17]: As above.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/0cb072686c12a8356ef3909b224829734be713c6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1aee36b9-a846-435d-bf02-c7bec68b287e) | 60  | 80  | 47  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/0cb072686c12a8356ef3909b224829734be713c6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | [72](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c0424cd9-b64c-40c9-99fa-f665da041a45) | 75  | 81  | 44  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.73cf202b-9e32-417f-84fb-80fe86a9e126?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/d97e8d0577050789f85871f2d7ae2004f637bc54/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | [70](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8647eedb-9c63-475f-abe8-19a4dfe4625f) | 60  | 80  | 47  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/d97e8d0577050789f85871f2d7ae2004f637bc54/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | [72](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a8572cc9-ce03-47ab-a33d-1b3e641a570d) | 75  | 81  | 44  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.a0225054-e668-40f6-b46c-04d028f9b363?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md (hunk 2 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023/index.md]: Yes, code font is a better idea. thanks!

**Pull request merged**